test/Validation.Tests/RequiresTests.cs

```diff
@@ -228,4 +230,22 @@ public void Defined_Int64Enum()
         InvalidEnumArgumentException ex = Assert.Throws<InvalidEnumArgumentException>(() => Requires.Defined((BigEnum)0x100000000, "parameterName"));
         this.logger.WriteLine(ex.Message);
     }
+
+    [Fact]
+    public void ValidElements()
+    {
+        Assert.Throws<ArgumentException>(() => Requires.ValidElements(new[] { -1 }, v => v > 0, "param", "Must be greater than 0."));
```

Comment:
I believe `Assert.Throws<T>` has an overload that takes an extra first argument that is the problematic parameter name that the argument exception is complaining about. Can we use those to verify we're putting the parameter name in the right argument of the `ArgumentException` constructor?
It may be good to verify the string formatting is correct too.

Reply:
It does. I will update the tests. Should I also update the other tests as well? None of them use that overload.

Reply:
Sure, if you're up for it. :)